src/emmc.c

```diff
@@ -615,9 +621,9 @@ pu_emmc_parse_partitions(PuEmmc *emmc,
             return FALSE;
         }
 
-        g_debug("Parsed partition: label=%s filesystem=%s type=%s size=%lld offset=%lld expand=%s",
+        g_debug("Parsed partition: label=%s filesystem=%s type=%s size=%lld offset=%lld align=%lld expand=%s",
```

Comment:
Same for this one. Breaking this string up into two lines would most certainly benefit its readability.